webapp/channels/src/utils/performance_telemetry/long_task.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/**
+ * See https://developer.mozilla.org/en-US/docs/Web/API/PerformanceLongTaskTiming
+ */
+export interface PerformanceLongTaskTiming extends PerformanceEntry {
```

Comment:
would updating node got us this type by default? If not node then which one

Reply:
No, it wouldn't. The definitions of `PerformanceEntry` and others actually come from TS, but `PerformanceLongTaskTiming` isn't in there because it's still an experimental API. Chrome added it in 2017 and published a specification for it, but it hasn't been adopted by any other browsers yet